configs/11.0/packages/gcc/stage_2

```diff
@@ -72,8 +72,9 @@ atcfg_pre_configure() {
 atcfg_configure() {
 	if [[ "${cross_build}" == "no" ]]; then
 		# Configure command for native builds
-		CC="${cc_64}" \
-		CXX="${cxx_64}" \
```

Comment:
You could clean the atcfg_pre_configure() to just set ${libdir} since we won't need ${c*_64} anymore.

Reply:
Done. gave a general cleanup on atcfg_pre_configure for stage 2 and 3 and removed the now unecessary ${c*_64} variables.